utils/src/ooinstall/oo_config.py

```diff
@@ -176,6 +177,10 @@ def _read_config(self):
                 if 'Description' in self.settings:
```

Comment:
If I remember right, we added the version field specifically to make this easier to find in the future.